students/wielgus_daniel/lesson_01_basics/lesson_01_part_02.py

```diff
@@ -0,0 +1,124 @@
+import math
+
+#Problem1: Last digit of integer
+print("We will show Last digit of integer")
+print("Please provide the digit:")
+a = int(input())
+b = str(a)
+print("The last digit is: " + b[-1])
+print("THE END of Last digit of integer problem")
+print("")
+
+#Problem2: Tens digit
+print("We will show Tens digit")
+print("Please provide the digit:")
+a = int(input())
+b = str(a)
+print("The Tens digit is: " + b[-2])
+print("THE END of Tens digit problem")
+print("")
+
+#Problem3: Sum of digits
+print("We will show sum of digit")
+print("Please provide the three-digit number:")
+number = int(input())
+strNumber = str(number)
+firstDigit = strNumber[-3]
+secondDigit = strNumber[-2]
+thirdDigit = strNumber[-1]
+sumOfDigits = int(firstDigit) + int(secondDigit) + int(thirdDigit)
+print("The sum of digits is: " + str(sumOfDigits))
```

Comment:
using `+` for formatting string is ok, but it's good to know that preferred way in python world is using `format`
Check it out at https://pyformat.info/